shared/src/main/scala/vyxal/Interpreter.scala

```diff
@@ -7,9 +7,8 @@ object Interpreter {
       code: String,
       inputs: List[VAny] = List.empty,
       flags: List[String] = List.empty
-  )(using Backend): Context = {
+  )(using Backend)(using ctx: Context): Context = {
```

Comment:
There's no need to take context as a parameter since flags are being passed in. The context should be built based on the flags, I just haven't implemented that yet.

Reply:
That was just for testing. I can remove it now.